packages/cadl-ranch-specs/http/lro/lro-rpc/util.tsp

```diff
@@ -0,0 +1,19 @@
+import "@azure-tools/typespec-azure-core";
+using Azure.Core;
+
+namespace LocalCore {
+  #suppress "@azure-tools/cadl-ranch-expect/missing-scenario" "Will use Azure.Core"
+  @doc("Long running RPC operation template")
+  op LongRunningRpcOperation<
+      TParams extends object,
+      TResponse extends object,
+      Traits extends object = {}
+    > is RpcOperation<
+      TParams & Traits,
+      Foundations.AcceptedResponse<Traits &
+                                  Foundations.LongRunningStatusLocation &
```

Comment:
It likely require another LongRunningStatusLocation definition, for the type of `TPollResponse`.

```ts
  @doc("Metadata for long running operation status monitor locations")
  model LongRunningStatusLocation<TPollResponse extends object> {
    @pollingLocation
    @doc("The location for monitoring the operation state.")
    @TypeSpec.Http.header("Operation-Location")
    operationLocation: Rest.ResourceLocation<TPollResponse>;
  }
```